python/tests/unit/arcticdb/version_store/test_sort_merge.py

```diff
@@ -868,3 +868,56 @@ def test_writing_wide_segment_over_sliced_data(self, lmdb_storage, lib_name):
         lib.sort_and_finalize_staged_data("sym", mode=StagedDataFinalizeMethod.WRITE)
 
         assert_frame_equal(lib.read("sym").data, df_1)
+
+
+@pytest.mark.parametrize("delete_staged_data_on_failure", [True, False])
+def test_sort_and_finalize_staged_data_write_dynamic_schema_named_index(
```

Comment:
nit: Can't this be parametrized by mode=[StagedDataFinalizeMethod.WRITE,, StagedDataFinalizeMethod.APPEND] as well to avoid repetition?